### ISSUE TITLE
Add support for multiple learning rates / weight decays

### ISSUE THREAD

**Issue by dlwh** [stanford-crfm/levanter#992]
in the same way we can configure weight decay masks, we should make it possible to specify different LRs for different components
